commands.go

```diff
@@ -1,7 +1,10 @@
 package main
 
 import (
+	pb "devchat/plugin"
 	"fmt"
+	"github.com/acarl005/stripansi"
+	"github.com/samber/lo"
```

Comment:
By convention stdlib imports come first and then everything else separated by a newline.